packages/taquito-signer/src/taquito-signer.ts

```diff
@@ -44,6 +51,27 @@ export class InMemorySigner {
     return new InMemorySigner(key, passphrase);
   }
 
+  /**
+   *
+   * @description Instantiation of an InMemorySigner instance from a mnemonic
+   * @param mnemonic 12-24 word mnemonic
+   * @param password password used to encrypt the mnemonic to seed value
+   * @param derivationPath default 44'/1729'/0'/0' (44'/1729' mandatory)
+   * @param curve currently only supported for tz1, tz2, tz3 addresses. soon bip25519
+   * @returns InMemorySigner
+   */
+  static fromMnemonic(mnemonic: string, password = '', derivationPath = "44'/1729'/0'/0'", curve?: Curves) {
```

Comment:
It might be unclear to the user that the default curve is "ed25519".
Consider adding the default value as part of the parameter (like for password and derivationPath) or adding a note in the comment.

Reply:
fixed the bad syntactic choice for a default value